docs/install/install-server/install-server.inc

```diff
@@ -64,25 +64,52 @@ Install the software
         First make sure Python >= 3.6 and git are installed. Inmanta requires many dependencies so it is recommended to create a virtual env.
         Next install inmanta with pip install in the newly created virtual env.
 
+        Please note, the path to the virtual env is arbitrary. Your desired path can override below example.
+
         .. code-block:: sh
 
             # Install python3 >= 3.6 and git
             sudo python3 -m venv /opt/inmanta
             sudo /opt/inmanta/bin/pip install inmanta
             sudo /opt/inmanta/bin/inmanta --help
+            
+            # Install PostgreSQL
+            sudo apt-get install postgresql postgresql-client
```

Comment:
This is in the section `Other Linux and Mac`, apt-get on mac is not going to work

Reply:
It might be good to add a ubuntu/debian section?

Reply:
It would. However, this `.inc` format is very confusing to me. Did not want to break something. IMHO, the Debian needs its own section since Centos is reaching its end of life.

Reply:
it is just rst format, use `.. tab::` to start another tab.

Reply:
ok